locales/en/plugin__odf-console.json

```
@@ -228,11 +228,24 @@
   "Application list": "Application list",
   "Last sync": "Last sync",
   "Activity status": "Activity status",
+  "Ready": "Ready",
+  "Not ready": "Not ready",
+  "Target cluster:": "Target cluster:",
+  "Last available: ": "Last available: ",
+  "{{actionType}} readiness:": "{{actionType}} readiness:",
+  "Data last synced on:": "Data last synced on:",
```

shouldn't this be ? `Data` is confusing, everywhere else we differentiate on basis of volume & kube objects...
```suggestion
  "Volume last synced on:": "Data last synced on:",
```